moveit_planners/stomp/stomp_moveit/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>stomp_moveit</name>
+  <version>0.0.0</version>
```

Comment:
match version to https://github.com/ros-planning/moveit/blob/indigo-devel/moveit/package.xml#L4